e2e/benchmarks/local-benchmark/index.js

```diff
@@ -169,6 +169,14 @@ async function initDefaultValueMap() {
   for (const flag in TUNABLE_FLAG_DEFAULT_VALUE_MAP) {
     state.flags[flag] = TUNABLE_FLAG_DEFAULT_VALUE_MAP[flag];
   }
+
+  // Remove it once we figure out how to correctly read the tensor data
+  // before the tensor is disposed in profiling mode.
+  if (tf.engine().backendNames().includes('webgpu')) {
+    tf.env().set('CHECK_COMPUTATION_FOR_ERRORS', false);
+    state.flags['CHECK_COMPUTATION_FOR_ERRORS'] = false;
```

Comment:
I think `initDefaultValueMap()` is only called once. You can't resolve that on other backend, the user changes `CHECK_COMPUTATION_FOR_ERRORS` to true again.
Maybe you should set it in `setEnvFlags`.

Reply:
You're right, I moved changes into showBackendFlagSettingsAndReturnTunableFlagControllers(), this function triggered with onChange event when backend changed and change CHECK_COMPUTATION_FOR_ERRORS to false when current backend is WebGPU.